Lesson_09/l09_h01_read_write_files.py

```diff
@@ -0,0 +1,33 @@
+"""
+HW:9[Volkov Anton].
+
+write script that will read "input.txt" file and find there all characters
+from english alphabet
+collect these letteracters and their positions in file
+after info collected this info as text
+write to "output.txt" file in such format
+####################
+<first found letter> -> pos1
+<next_letter> -> pos2
+<next_letter -> pos3
+etc
+#######################
+"""
+import re
+
+SEPARATOR = '#' * 15 + '\n'
+with open('input.txt', 'r') as fh:
```

Comment:
Better to use comments for certain block - for example here could be comment that we read from files alphabet letter and their positions. Then in output block we able to leave comment that we write letter -> pos to file
Not critical but your code visually will be divided into logical blocks that will simplify code reading